mfthomps/CyberCIEGE-port-to-Unity3d
# Merge Request for Issue/136 - Component View Asset tab

Closes #136 

To test:
- Run any scenario with multiple computers in it
- Open Component View and click Assets tab
- Select computer to see what assets are assigned to it
- Click asset in "Assigned To This Computer" to unassign it
- Click asset in "Not Assigned To This Computer" to assign it
- Make sure to try assigning an asset that is already assigned to another computer and make sure it is only assigned to the new one
